plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/nessie/TestTrinoNessieCatalog.java

```diff
@@ -86,12 +85,10 @@ protected TrinoCatalog createTrinoCatalog(boolean useUniqueTableLocations)
             fail(e.getMessage());
         }
         TrinoFileSystemFactory fileSystemFactory = new HdfsFileSystemFactory(HDFS_ENVIRONMENT, HDFS_FILE_SYSTEM_STATS);
-        IcebergNessieCatalogConfig icebergNessieCatalogConfig = new IcebergNessieCatalogConfig()
-                .setServerUri(URI.create(nessieContainer.getRestApiUri()));
         NessieApiV1 nessieApi = HttpClientBuilder.builder()
                 .withUri(nessieContainer.getRestApiUri())
                 .build(NessieApiV1.class);
-        NessieIcebergClient nessieClient = new NessieIcebergClient(nessieApi, icebergNessieCatalogConfig.getDefaultReferenceName(), null, ImmutableMap.of());
+        NessieIcebergClient nessieClient = new NessieIcebergClient(nessieApi, null, null, ImmutableMap.of());
```

Comment:
Why?

Reply:
The code to create `icebergNessieCatalogConfig`  was superfluous. Hence, I optimized it.  
Since it is slightly unrelated to the PR changes, I reverted it now.